policybot/pkg/gh/fromhook.go

```diff
@@ -52,10 +55,13 @@ func IssueFromHook(ip *hook.IssuesPayload) *storage.Issue {
 		State:       ip.Issue.State,
 		AuthorID:    ip.Issue.User.NodeID,
 		AssigneeIDs: assignees,
-	}
+	}, discoveredUsers
```

Comment:
nit: splitting create and return into different statements for better readability.